README.md

```diff
@@ -17,7 +17,27 @@ A gem for easily using read replicas.
 
 ## Supported Versions
 
-Knockoff supports Rails 4, 5 and 6
+| Knockoff version: | 1.0 | 1.1.0 | 1.1.1 | 1.4 | 1.5 |
+| Ruby 2.2 | [x] | [ ] | [ ] | [ ] | [ ] |
+| Ruby 2.3 | [x] | [x] | [ ] | [ ] | [ ] |
+| Ruby 2.4 | [x] | [x] | [x] | [ ] | [ ] |
+| Ruby 2.5 | [ ] | [x] | [x] | [ ] | [ ] |
+| Ruby 2.6 | [ ] | [ ] | [x] | [x] | [ ] |
+| Ruby 2.7 | [ ] | [ ] | [x] | [x] | [x] |
+| Ruby 3.0 | [ ] | [ ] | [ ] | [x] | [x] |
+| Ruby 3.1 | [ ] | [ ] | [ ] | [x] | [x] |
+
+| Knockoff version: | 1.0 | 1.1.0 | 1.1.1 | 1.4 | 1.5 |
+| Rails 4 | [x] | [x] | [ ] | [ ] | [ ] |
+| Rails 5.0 | [x] | [x] | [x] | [ ] | [ ] |
+| Rails 5.1 | [x] | [x] | [x] | [ ] | [ ] |
+| Rails 5.2 | [x] | [x] | [x] | [ ] | [ ] |
+| Rails 6.0 | [ ] | [ ] | [x] | [x] | [ ] |
+| Rails 6.1 | [ ] | [ ] | [ ] | [x] | [x] |
+| Rails 7.0 | [ ] | [ ] | [ ] | [ ] | [x] |
+| Rails 7.1 | [ ] | [ ] | [ ] | [ ] | [ ] |
+| Rails 7.2 | [ ] | [ ] | [ ] | [ ] | [ ] |
+
```

Comment:
This is my best guess, it didn't seem worth figuring it out exactly since no one has been very careful about documenting releases

Reply:
it's missing the header line to make it a table 

```
| one | two | three |
|---|---|---|
| foo | bar | baz |
```